### DISCUSSION TITLE
esp32 cam matter

### DISCUSSION THREAD

**Discussion by montyfert** [arendst/Tasmota#19949] [Q&A] [answered]
hi are there any plans to add matter to the tasmota esp32 camera bin.
thanks 
martin

**Comment by sfromis** [comment, marked as answer]
With the camera function already a pretty tall order for the EPS32, this build is deliberately "kept low" in other features, as many things would just not be a good combination with streaming video.

**Reply by montyfert** [reply]
ok thanks for the quick reply 
martin
